tests/config/config.d/merge_tree_settings.xml

```diff
@@ -2,5 +2,6 @@
     <merge_tree>
         <!-- 10 seconds (default is 1 minute) -->
         <zookeeper_session_expiration_check_period>10</zookeeper_session_expiration_check_period>
+        <allow_experimental_block_number_column>0</allow_experimental_block_number_column>
```

Comment:
We should probably leave a comment here about why it's disabled for now

Reply:
addressed it